### DISCUSSION TITLE
This is getting SO popular, but were not done finishing this language yet.

### DISCUSSION THREAD

**Discussion by Rubiidev-18** [cytrinc/bark-coding#42] [General]
https://s01.flagcounter.com/more30/z8GL/
![Screenshot_20240321_173848429Quick](https://github.com/mariocraft987/bark.github.io/assets/158863401/e2ebe1f3-05e9-4ca1-9aa2-33affe8419a1)

**Comment by mariocraft987** [comment]
Or maybe people explore different urls.... Probably

**Comment by raynepaws** [comment]
does it count the same ip address every time or just once? that could be what's happening here

**Reply by nmsderp** [reply]
it doesn't, but if you use a VPN it would

**Comment by mariocraft987** [comment]
Idk
